templates/operations.ts.hbs

```diff
@@ -115,6 +117,7 @@
         body: {{{getPayloadTypeFromRequest request}}}{{/or}}
       }>,
       rawResponse?: boolean,
+      throwOnBadResponse?: boolean
```

Comment:
Throwing on bad responses is probably a behavior that the SDK user wants to be consistent every time.  This should be set on the client config, rather than being passed every single time they call a method.